### ISSUE TITLE
Using maskedInput plugin problem

### ISSUE THREAD

**Issue by SigalZ** [formvalidation/support#171]
Hi,

I am using maskedInput jquery plugin to mask a phone field.

I am also using your mandatory icon addon.

The mandatory icon does not appear on the phone field and the field is not being validated.

My code:
`$('#registerForm').formValidation('destroy')
        .formValidation({
            framework: 'bootstrap',
            excluded: ':disabled',
            icon: {
                valid: 'glyphicon glyphicon-ok',
                invalid: 'glyphicon glyphicon-remove',
                validating: 'glyphicon glyphicon-refresh'
            },
            addOns: {
                mandatoryIcon: {
                    icon: 'glyphicon glyphicon-asterisk'
                }
            },
            fields: {
                customer_first_name: {
                    validators: {
                        notEmpty: { message: "Please enter your first name" }
                    }
                },
                customer_surname: {
                    validators: {
                        notEmpty: { message: "Please enter your surname" }
                    }
                },
                customer_email: {
                    trigger: 'blur',
                    validators: {
                        notEmpty: { message: "Please enter your email" },
                        emailAddress: { message: "The email address you entered is not valid"},
                        duplicateEmail: { message: "The email address you entered already exists"}
                    }
                },
                customer_phone: {
                    validators: {
                        notEmpty: { message: "Please enter your phone number" }
                    }
                },
                customer_pass: {
                    validators: {
                        notEmpty: { message: "Please enter a password" },
                        securePassword: { message: 'The password is not valid' }
                    }
                },
                b_addr_line_1: {
                    validators: {
                        notEmpty: { message: "Please enter your billing address line 1" }
                    }
                },
                b_city: {
                    validators: {
                        notEmpty: { message: "Please enter your billing city" }
                    }
                },
                b_province: {
                    validators: {
                        callback: {
                            message: 'Please select your billing province',

                            callback: function(value, validator, $field) {
                                if($field.val() == '') {
                                    $('#bProvinceDiv').find($('.select2-selection--single')).css('border', '1px solid #a94442');
                                } else {
                                    $('#bProvinceDiv').find($('.select2-selection--single')).css('border', '1px solid #aaa');
                                }

                                return ($field.val() != '');
                            }
                        }
                    }
                },
                b_post_code: {
                    validators: {
                        notEmpty: { message: "Please enter your billing post code" }
                    }
                },
                s_addr_line_1: {
                    validators: {
                        notEmpty: { message: "Please enter your shipping address line 1" }
                    }
                },
                s_city: {
                    validators: {
                        notEmpty: { message: "Please enter your shipping city" }
                    }
                },
                s_province: {
                    validators: {
                        callback: {
                            message: 'Please select your shipping province',

                            callback: function(value, validator, $field) {
                                if($field.val() == '') {
                                    $('#sProvinceDiv').find($('.select2-selection--single')).css('border', '1px solid #a94442');
                                } else {
                                    $('#sProvinceDiv').find($('.select2-selection--single')).css('border', '1px solid #aaa');
                                }

                                return ($field.val() != '');
                            }
                        }
                    }
                },
                s_post_code: {
                    validators: {
                        notEmpty: { message: "Please enter your shipping post code" }
                    }
                }
            }
        })
        .find('[name="customer_phone"]').mask('(999) 999-9999')
        .on('success.form.fv', function(e) {
            // Prevent form submission
            e.preventDefault();

            var $form = $(e.target);

            // Use Ajax to submit form data
            $form.ajaxSubmit({
                url: '/modules/customers/ajax/register.ajax.php',
                type: 'POST',
                //data: $form.serialize(),
                beforeSubmit: function() { alert('wy');
                    $('.ajaxLoading').show();
                },
                success: function(result) {
                    $('.ajaxLoading').hide();
                    bootbox.dialog({
                        message: "Thank you for registering on our site.<br>An activation email has been sent to the email address you supplied.",
                        title: "<span class='dialogSuccess'>Success</span>",
                        buttons: {
                            exit: {
                                label: "Exit",
                                className: "btn-success",
                                callback: function() {
                                    $('#registerDiv').slideUp();
                                    $('#login').slideDown();
                                    $('#registerForm')[0].reset();
                                    $('#registerForm').data('formValidation').resetForm();
                                    $('.list').select2('val', '');
                                }
                            }
                        }
                    });
                }
            });
        });`

My html:
`<form name="registerForm" method="post" id="registerForm">
                    <div class="form-group">
                        <lable for="firstName">First Name</lable>
                        <input type="text" name="customer_first_name" id="firstName" class="form-control">
                    </div>
                    <div class="form-group">
                        <lable for="surname">Surname</lable>
                        <input type="text" name="customer_surname" id="surname" class="form-control">
                    </div>
                    <div class="form-group">
                        <lable for="customerEmail">Email (Will be used for your login)</lable>
                        <input type="text" name="customer_email" id="customerEmail" class="form-control">
                    </div>
                    <div class="form-group">
                        <lable for="phone">Phone Number</lable>
                        <input type="text" name="customer_phone" id="phone" class="form-control phone">
                    </div>
                    <div class="form-group">
                        <lable for="pass">Password</lable>
                        <input type="password" name="customer_pass" id="pass" class="form-control">
                    </div>
Addresses fields coming here.....
`

**Comment by SigalZ** [comment]
Ok, found the solution, I used a different plugin, it works now.
